doc/source/working_with_kiwi.rst

```diff
@@ -84,3 +85,149 @@ to find in the *description directory*:
           <archive name="custom-archive.tgz"/>
       </packages>
 
+
+.. _working-with-kiwi-image-building-process:
+
+Image Building Process
+----------------------
+
+KIWI creates images in a two step process. The first step, the *prepare*
+operation, generates a so-called *unpacked image* tree (directory) using
+the information provided in the :file:`config.xml` configuration file
+(see :ref:`xml-description`)
+
+The second step, the *create* operation, creates the *packed image* or
+*image* in the specified format based on the unpacked image and the
+information provided in the :file:`config.xml` configuration file.
+
+.. figure:: .images/intro.png
+    :align: center
+    :alt: Image Creation Architecture
+
+    Image Creation Architecture
+
+
+.. _prepare-step:
+
+The Prepare Step
+^^^^^^^^^^^^^^^^
+
+The first step of the image creation is the *prepare* step, during which
```

Comment:
I think you can simplify the sentence (as you are already talking about the prepare step):

```suggestion
As the first step, KIWI creates an *unpackaged image tree*, also called "root tree". This
```